backend/internal/sessionsgrpcsvc/svc.go

```diff
@@ -32,27 +32,36 @@ func Init(mux *http.ServeMux, sessions sdkservices.Sessions) {
 	mux.Handle(path, handler)
 }
 
+// re-wrap sdk as connect error
+func wrapError(err error, code connect.Code) error {
```

Comment:
don't we have a place that already does something like that for all errors ?
and if not, this looks like a good candidate to have in a shared place

Reply:
sure. I think we should open a ticket for refactoring @itayd ? @daabr ?

Reply:
added a ticket. resolving